src/config.rs

```diff
@@ -675,6 +675,8 @@ impl DefaultCfConfig {
         cache: &Option<Cache>,
         region_info_accessor: Option<&RegionInfoAccessor>,
         api_version: ApiVersion,
+        region_id: u64,
```

Comment:
Looks a bit ugly, but I can't think of a better way either. cc @BusyJay

Reply:
I suggest to move compaction filter generation to the `KvEngineFactory` instead of configuration.

Reply:
ok. I took your suggestion.